cmd/update.go

```diff
@@ -69,6 +86,13 @@ func newUpdateCommand(config *settings.Config) *cobra.Command {
 		},
 	})
 
+	update.PersistentFlags().BoolVar(&opts.noTTY, "no-tty", false, "Enable non-interactive mode for clients without a terminal")
+
```

Comment:
Since we're hiding `update check` maybe it makes sense to add a flag to do this and set `dryRun`? `update --check` ?

Reply:
I've added `--check` in cf51a40.

Reply:
I've added `--check` in cf51a40.

Reply:
I've added `--check` in cf51a40.